modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js

```diff
@@ -25,7 +25,6 @@ class DDMFormHandler {
 		this.fields = getDefaultFieldsShape(DDMFormInstance);
 
 		this._attachFormListener();
-		this.checkCPInstance();
```

Comment:
Hey @alecsloan are we sure we want to remove this? As far as I remember in the very first implementation I had not run this first check either. But then "someone" made me figure out that if we have initial combos of options in the form, we may want to pick the right product based on those and notify the other components accordingly. This wouldn't happen without this first run of `this.checkCPInstance()`.

Did this change in the backend so that we pick right away the right product based on the initial options configuration?